artcommon/artcommonlib/util.py

```diff
@@ -34,6 +34,15 @@ def remove_suffix(s: str, suffix: str) -> str:
         return s[:]
 
 
+def yaml_handler():
```

Comment:
This function needs more describable name, like `new_roundtrip_yaml_processor`

Reply:
updated